monkey/infection_monkey/credential_collectors/ssh_collector/ssh_handler.py

```diff
@@ -165,7 +165,7 @@ def to_credentials(ssh_info: Iterable[Dict]) -> Sequence[Credentials]:
 
 def _publish_credentials_stolen_event(collected_credentials: Credentials, event_queue: IEventQueue):
     credentials_stolen_event = CredentialsStolenEvent(
-        tags=frozenset(SSH_COLLECTOR_EVENT_TAG),
+        tags=frozenset(SSH_COLLECTOR_EVENT_TAGS),
```

Comment:
See comments on the mimikatz collector. Just make `SSH_COLLECTOR_EVENT_TAGS` a frozenset.